packages/terra-graphs-docs/src/terra-dev-site/doc/example/Graphs-Carbon/General/Axes/XAxisTicksFormatted.jsx

```diff
@@ -3,24 +3,25 @@ import Carbon from '@cerner/carbon-graphs/lib/js/carbon';
 import utils from '@cerner/carbon-graphs/lib/js/helpers/utils';
 import '@cerner/terra-graphs-docs/lib/terra-graphs-src/components/Graph.module.scss';
 import ExampleGraphContainer from '@cerner/terra-graphs-docs/lib/terra-dev-site/ExampleGraphContainer/ExampleGraphContainer';
-import lineTimeseries from '@cerner/terra-graphs-docs/lib/example-datasets/graphConfigObjects/Line/lineTimeseries';
+import getGraphConfig from '@cerner/terra-graphs-docs/lib/example-datasets/graphConfigObjects/General/timeseriesConfigTicksFormatted';
 
-const tickValues = [
-  new Date(2016, 0, 1, 1, 0).toISOString(),
-  new Date(2016, 0, 1, 5, 0).toISOString(),
-  new Date(2016, 0, 1, 10, 0).toISOString(),
-  new Date(2016, 0, 1, 15, 0).toISOString(),
-  new Date(2016, 0, 1, 20, 0).toISOString(),
-];
+// const tickValues = [
+//   new Date(2016, 0, 1, 1, 0).toISOString(),
+//   new Date(2016, 0, 1, 5, 0).toISOString(),
+//   new Date(2016, 0, 1, 10, 0).toISOString(),
+//   new Date(2016, 0, 1, 15, 0).toISOString(),
+//   new Date(2016, 0, 1, 20, 0).toISOString(),
+// ];
```

Comment:
Remove comments. Looks like there are comments in other places too, please remove all of them if not necessary.